sdks/python/setup.py

```diff
@@ -121,7 +119,7 @@ def get_version():
     ]
 
 GCP_REQUIREMENTS = [
-  'google-apitools>=0.5.10,<=0.5.11',
```

Comment:
Have you tested this by running a job on Dataflow?

This is a little scary because the latest version of apitools on github is 0.5.14.

@craigcitro @charlesccychen @chamikaramj (FYI, in case anybody is aware of a known incompatibility.)

Reply:
Python SDK PostCommit Tests have successfully completed, which use TestDataflowRunner.

Reply:
Awesome! Thank you.